From 0f8100aeda8d04a604319b98108b5d331fa161c3 Mon Sep 17 00:00:00 2001
From: Fabio Berton <fabio.berton@ossystems.com.br>
Date: Mon, 2 Jul 2018 17:09:38 -0300
Subject: [PATCH] rules.d/90-stick.rules: Change rule to use with UpdateHub
Organization: O.S. Systems Software LTDA.

Change to match flash drive with name smaai5upd and SMAAI5UPD for
smaai5 product.

Signed-off-by: Fabio Berton <fabio.berton@ossystems.com.br>
---
 stick/etc/udev/rules.d/90-stick.rules | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/stick/etc/udev/rules.d/90-stick.rules b/stick/etc/udev/rules.d/90-stick.rules
index c803506..40f4b62 100644
--- a/stick/etc/udev/rules.d/90-stick.rules
+++ b/stick/etc/udev/rules.d/90-stick.rules
@@ -1,2 +1,2 @@
-SUBSYSTEM=="block", ACTION=="remove", KERNEL=="sd*", NAME="%k", RUN+="/usr/lib/stick/stick-umount.sh"
-SUBSYSTEM=="block", ACTION=="add", KERNEL=="sd?1", NAME="%k", RUN+="/usr/lib/stick/stick-mount.sh"
+ENV{ID_FS_LABEL}!="smaai5upd|SMAAI5UPD", ENV{DEVTYPE}=="partition", ACTION=="remove", KERNEL=="sd*", NAME="%k", RUN+="/usr/lib/stick/script/sh/stick-umount.sh"
+ENV{ID_FS_LABEL}!="smaai5upd|SMAAI5UPD", ENV{DEVTYPE}=="partition", ACTION=="add", KERNEL=="sd?1", NAME="%k", RUN+="/usr/lib/stick/script/sh/stick-mount.sh"
-- 
2.18.0

